lib/internal/Magento/Framework/Shell/CommandRendererBackground.php

```diff
@@ -33,8 +44,14 @@ public function render($command, array $arguments = [])
     {
         $command = parent::render($command, $arguments);
 
+        $logFile = '/dev/null';
+        if ($groupId = $arguments[2] ?? null) {
```

Comment:
As per discussion in the guild, $arguments[2] is not supposed to $group_id always, since it’s a general class that might be used not only by cron:run command.

Reply:
Based on this, I have removed the modifications to the Framework classes.